src/CarouselItem.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Transition, { ENTERING, ENTERED, EXITING } from 'react-transition-group/Transition';
```

Comment:
Looks like you are removing these enums to better support when ReactTransitionGroup is not included, but can you put these in the utils and reference them from there?

Reply:
Yes, we can move these to utils. The main reason is that since we are cherry picking an export it prevents using the full ES6 capability for the module in production. (This gives reduced file size in the now `reactstrap.full.min.js`; however, at the expense of not being able to use these.